x-pack/test/security_solution_endpoint_api_int/apis/resolver/common.ts

```diff
@@ -59,12 +396,13 @@ const expectLifecycleNodeInMap = (
 /**
  * Verify that all the ancestor nodes are valid and optionally have parents.
  *
+ * @deprecated use verifyTree
  * @param ancestors an array of ancestors
  * @param tree the generated resolver tree as the source of truth
  * @param verifyLastParent a boolean indicating whether to check the last ancestor. If the ancestors array intentionally
  *  does not contain all the ancestors, the last one will not have the parent
  */
-export const verifyAncestry = (
+export const checkAncestryFromEntityTreeAPI = (
```

Comment:
Renaming most of the functions to indicated that they're for the old tree api route centered around an entity_id